src/Database/LSMTree/Internal/BloomFilter.hs

```diff
@@ -125,3 +129,57 @@ hGetByteArrayExactly hfs h len = do
     _   <- hGetBufExactly hfs h buf 0 (fromIntegral len)
     P.unsafeFreezeByteArray buf
 
+-- Bulk insert
+-----------------------------------------------------------
+
+{-# INLINE bloomInserts #-}
+bloomInserts ::
+     forall a s.
+     BF.Hashable a
+  => BF.MBloom s a
+  -> (Int -> ST s a)
+  -> Int
+  -> ST s ()
+bloomInserts bloom key n =
+    P.newPrimArray 0x10 >>= body
```

Comment:
So prefetch distance 15 or 16 depending on how you define prefetch distance, right?

Reply:
Exactly.

Reply:
Documented.